sequencing-server/tsconfig.json

```diff
@@ -1,7 +1,14 @@
 {
   "extends": "@tsconfig/node16-strictest-esm/tsconfig.json",
-  "exclude":["sequencing_file_store", "build","cdict"],
+  "exclude": [
+    "sequencing_file_store",
+    "build",
+    "cdict"
+  ],
   "compilerOptions": {
-    "lib": ["es2022"]
+    "lib": [
+      "esnext"
```

Comment:
Any issues using `esnext`? It is essentially the next version before becoming es2023 whenever that gets finalized. 

Have you tried doing the below instead to import the schema:

```
"compilerOptions": {
    "resolveJsonModule" : true,
    "lib": ["es2022"]
  }
```

Reply:
I haven't seen any, yeah I originally tried with `es2022` but it didn't work for the schema import.

Reply:
Even with`"resolveJsonModule" : true`

Reply:
That's surpising to me because I have definitely done the json module resolution with versions far prior to es2022.

Reply:
Does it work without the import assertion (`assert { type: 'json' }`)?

Reply:
Yeah, I added `"resolveJsonModule": true` first before trying `esnext`.

Reply:
Import assertions are notoriously poorly supported.

Reply:
Have you tried removing the import assertion and using `"esModuleInterop": true` as well?

Reply:
Yeah I've def done json imports on past TypeScript versions as well. Not sure why this is not working for `es2022`.

Reply:
we can always use // @ts-ignore :P

Reply:
I guess this is the only option as of now so we can Approve. Just keep a look out for when es2023 is released so we can switch to that version if it comes out. Not sure how these releases work.